include/MySQL_Monitor.hpp

```diff
@@ -51,6 +51,26 @@ struct cmp_str {
 #define AWS_Aurora_Nentries	50
 
 #define N_L_ASE 16
+
+/**
+ * Enum holding the different connection errors that could be reached for
+ * the backend connections oppened by 'MySQL_Monitor' module.
+ */
+enum PROXYSQL_MONITOR_ERR {
```

Comment:
Any reason why we have `PROXYSQL_MONITOR_ERR` and not just extend `PROXYSQL_MYSQL_ERR` ?

Reply:
I thought it could be handy to have a clear separation between general errors and monitoring errors, easier to remember error code ranges that particular values.